src/coreclr/jit/codegen.h

```diff
@@ -208,6 +208,7 @@ class CodeGen final : public CodeGenInterface
     void**    codePtr;
     uint32_t* nativeSizeOfCode;
     unsigned  codeSize;
+    size_t    dataSize;
```

Comment:
This should probably live with the other related fields in [compiler.h](https://github.com/dotnet/runtime/blob/79d8dadf915d7d13b0aee3933c5b3d51a4940159/src/coreclr/jit/compiler.h#L11226).

Reply:
Hm, okay, i moved define dataSize in compiler.h
Thank you!